tests/evaluate/test_evaluation.py

```diff
@@ -361,6 +363,73 @@ def baseline_model_uri(request):
     return None
 
 
+def test_mlflow_evaluate_logs_traces():
+    TRACE_BUFFER.clear()
+    eval_data = pd.DataFrame(
+        {
+            "inputs": [
+                "What is MLflow?",
+                "What is Spark?",
+            ],
+            "ground_truth": ["What is MLflow?", "Not what is Spark?"],
+        }
+    )
+
+    def model(inputs):
+        return inputs
+
+    evaluate(model, eval_data, targets="ground_truth", extra_metrics=[mlflow.metrics.exact_match()])
+    assert len(get_traces()) == 1
+    TRACE_BUFFER.clear()
+
+
+def test_model_uri_evaluate_logs_traces(multiclass_logistic_regressor_model_uri, iris_dataset):
+    TRACE_BUFFER.clear()
+    evaluate(
+        multiclass_logistic_regressor_model_uri,
+        iris_dataset._constructor_args["data"],
+        model_type="classifier",
+        targets=iris_dataset._constructor_args["targets"],
+        evaluators="dummy_evaluator",
+    )
+    assert len(get_traces()) == 1
```

Comment:
What does the trace look like with a regressor model? In which scenario would the user want to see a trace for a regressor evaluation?

Reply:
Hi @jessechancy , is this addressed somewhere? Could you share a pointer if possible?

Besides, I'm wondering if we should provide a config to turn off the tracing for evaluate() runs, in case non-genai users get confused with the auto-logged traces. cc @BenWilson2